### PR TITLE
KB - immutable StorageClass object

### PR DESCRIPTION
**Pull request by hubvu**

### CODE REVIEW

File: docs/install/Advanced/_index.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ description: >
 
 ## Overview
 
-The guides below are recommended for users who are experiened in deploying infrastructure and applications in constrained environments and following [Infrastructure as Code (IaC)](https://en.wikipedia.org/wiki/Infrastructure_as_code) princples. 
\ No newline at end of file
+The guides below are recommended for users who are experiened in deploying infrastructure and applications in constrained environments and following [Infrastructure as Code (IaC)](https://en.wikipedia.org/wiki/Infrastructure_as_code) princples.
```
Comment (cannischan) [inline docs/install/Advanced/_index.md:11]: principles

File: docs/knowledgebase/solution-unable-to-apply-paramters-to-an-existing-storageclass-object.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Solution - Unable To Apply Parameters To An Existing StorageClass Object"
+linkTitle: "Solution - Unable To Apply Parameters To An Existing StorageClass Object"
+---
+
+## Issue
+
+You have created a custom `StorageClass` object in your Ondat cluster but you are experiencing an issue where you cannot interactively edit or patch the said `StorageClass` object and apply a custom `StorageClass` parameter to enable specific Ondat [features](/docs/concepts/labels/).
```
Comment (cannischan) [inline docs/knowledgebase/solution-unable-to-apply-paramters-to-an-existing-storageclass-object.md:8]: object **or** apply a custom..

**Review by cannischan — COMMENTED** [review]
small changes needed

File: docs/knowledgebase/solution-unable-to-apply-paramters-to-an-existing-storageclass-object.md (hunk 2 above)
Comment (hubvu) [reply docs/knowledgebase/solution-unable-to-apply-paramters-to-an-existing-storageclass-object.md:8]: I think "to" is the better choice here. I will change it.

File: docs/install/Advanced/_index.md (hunk 1 above)
Comment (hubvu) [reply docs/install/Advanced/_index.md]: fixed

**Review by cannischan — DISMISSED** [review]
lgtm

**Review by cannischan — APPROVED** [review]
lgtm

**Pull request merged**